apps/keyboard/js/layouts/fa.js

```diff
@@ -0,0 +1,98 @@
+Keyboards.fa = {
+  label: 'Persian',
+  shortLabel: 'فا',
+  menuLabel: 'فارسی',
+  secondLayout: true,
+  specificCssRule: true,
+  types: ['text', 'url', 'email'],
+  alternateLayoutKey: '۱۲۳',
+  basicLayoutKey: 'اب‌پ',
+  width: 11,
+  lang: 'fa',
+  needsCommaKey: true,
+  alt : {
+    'ض': ['۱', '1'],
```

Comment:
Actually, since there isn't any number keys on the first page, defining "alt" for these keys has no effect.